tests/pytests/test_wfn.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+import numpy as np
+import psi4
+
+from utils import compare_arrays
+
+pytestmark = [pytest.mark.psi, pytest.mark.api]
+
+def test_export_ao_elec_dip_deriv():
```

Comment:
Also, is this function even run by the test suite?

Reply:
1. I forgot to rename this test after I copy-pasted from another file. Thanks for the catch, fixed!
2. My experience has been that `pytest` will pick up anything as long as the file name and function name start with `test_`. @loriab would know for sure.

Reply:
Yes, since we started globbing (https://github.com/psi4/psi4/blob/master/psi4/CMakeLists.txt#L302), this will get run. It's not quick or smoke, so it won't get run in present CI though.

Reply:
I can mark this as quick if you want, since this is just DZ H2O.

Reply:
ok, sure. if you do, remove the `#TEST` (doesn't do anything for pytests), and should the `compare(ref, computed, ...` be switched?